helm/mds/templates/processor-deployment.yaml

```diff
@@ -0,0 +1,109 @@
+{{- range $name, $processor := .Values.processors }}
+  {{- if $processor.enabled }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $name }} # beter name
+  labels:
+    app: {{ $name }}
+  namespace: {{ $.Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ $name }}
+  template:
+    metadata:
+      labels:
+        app: {{ $name }}
+    spec:
+      {{- if (eq $name "mds-config") }}
+      volumes:
+        - name: mds-config-files
+          configMap:
+            name: mds-config-files
+      {{- end }}
```

Comment:
I don't think this is necessary

Reply:
It's not currently used but config will be needed by this processor down the line